### PR TITLE
Feature/cluster create adjust

### PR DESCRIPTION
**Pull request by WeiXuSeu**
#### What type of this PR

Add one of the following kinds:
/kind feature
adjuct cluster create logic, monitor pipeline result, get cluster meta info for cluster creation

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

<h4 style="box-sizing: border-box; margin-bottom: 16px; margin-top: 24px; font-weight: 600; font-size: 14px; line-height: 1.25; color: rgb(36, 41, 46); font-family: -apple-system, system-ui, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">ChangeLog</h4>

Language | Changelog
-- | --
🇺🇸 English | adjuct cluster create logic, monitor acture cluster create pipeline result, get cluster meta info for cluster creation
🇨🇳 中文 | 调整集群创建逻辑，在 cmp 模块中定时监测集群创建 pipeline 的结果，获取创建集群的元信息，并更新集群信息

#### Specified Reviewers:

/assign @luobily 

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ FROM ${BASE_DOCKER_IMAGE} as build
 RUN mkdir -p "$GOPATH/src/github.com/erda-project/erda/"
 COPY . "$GOPATH/src/github.com/erda-project/erda/"
 WORKDIR "$GOPATH/src/github.com/erda-project/erda/"
+ENV GOPROXY https://goproxy.cn/,direct
```
Comment (liuhaoyang) [inline build/dockerfiles/Dockerfile:8]: Why add goproxy proxy ?

**Review by liuhaoyang — CHANGES_REQUESTED** [review]

File: build/dockerfiles/Dockerfile (hunk 1 above)
Comment (WeiXuSeu) [reply build/dockerfiles/Dockerfile:8]: Because when build image locally, it cannot download pkg sometimes without proxy.

Comment (WeiXuSeu) [reply build/dockerfiles/Dockerfile:8]: It may affect image build in hongkong, so removed it.

**Review by liuhaoyang — APPROVED** [review]

**Pull request merged**